spec/factories/leagues.rb

```diff
@@ -2,10 +2,5 @@
   factory :league, parent: :event, traits: [:has_dates], class: League do
     game_mode :round_robin
     gameday_duration 7
-
-    factory :league_with_teams, class: League, parent: :event_with_teams, traits: [:has_dates] do
```

Comment:
why are you removing this?

Reply:
Because you can just use the new trait :with_teams instead and combine features of event objects as you wish

Reply:
Ah I got it. Pretty sweet with new trait.